provider/win_dump_utils.py

```diff
@@ -136,12 +136,12 @@ def dump_windbg_check(test, params, session):
                                timeout=480, step=10):
         test.error("Cannot generate dump analyze log.")
     chk_id_cmd = params["chk_id_cmd"] % log_file
-    status, output = session.cmd_status_output(chk_id_cmd)
-    if status:
+    if utils_misc.wait_for(lambda: not session.cmd_status(chk_id_cmd),
```

Comment:
L135 is also dump file analysis log existed check, right?

Reply:
yes

Reply:
So seems that it's not the root cause of that issue.

Reply:
Why?
L135 is used to check if the file is existd,this patch's modifiication is use to check the full log.Because the log need need some time to generate.

Reply:
Could you try to re-run the test case with this patch?

Reply:
> Why? L135 is used to check if the file is existd,this patch's modifiication is use to check the full log.Because the log need need some time to generate.

Got your point.

Reply:
Okay!